### ISSUE TITLE
Local functions: Change signature doesn't update the signature of the local function

### ISSUE THREAD

**Issue by jasonmalinowski** [dotnet/roslyn#8359]
1. Paste this into a new file:

``` C#
        static void Main(string[] args)
        {
            void Local(int x, int y)
            {
            }

            Local(1, 2);
        }
```
1. Invoke change signature on the local function.
2. Reorder the two parameters.

_Expected:_ the parameters of Local are reordered
_Actual:_ the parameters aren't reordered. Call sites are correctly updated.

(FYI @dpoeschl)

**Comment by balajikris** [comment]
I'm seeing something different. I'm unable to bring up change signature on any local function. Not via light bulb or the remove/reorder parameters command.

**Comment by jcouv** [comment]
Fix went in for 15.7 preview 3.
